base/tuple.jl

```diff
@@ -389,15 +389,17 @@ reverse(t::Tuple) = revargs(t...)
 
 ## specialized reduction ##
 
-# TODO: these definitions cannot yet be combined, since +(x...)
-# where x might be any tuple matches too many methods.
-# TODO: this is inconsistent with the regular sum in cases where the arguments
```

Comment:
This comment (the second TODO) is still relevant --- it refers to e.g. `sum(Int8[1])` vs. `sum((Int8(1),))`.

Reply:
Thanks. I'll add back and clarify the TODO.